docs/nuxt.config.js

```diff
@@ -1,18 +1,18 @@
 import theme from '@nuxt/content-theme-docs'
 
 export default theme({
-  loading: { color: '#00CD81' },
-  i18n: {
-    locales: () => [{
-      code: 'en',
-      iso: 'en-US',
-      file: 'en-US.js',
-      name: 'English'
-    }],
-    defaultLocale: 'en'
+  docs: {
+    primaryColor: '#AA7AB5'
   },
-  content: {
-    liveEdit: false
+  buildModules: ['nuxt-ackee'],
+  ackee: {
```

Comment:
Idea: this can be moved under docs: options to automatically add ackee

Reply:
Indeed, this will be another refactor 👍